README.md

```diff
@@ -1,3 +1,101 @@
-# CTD Omega Services
+# Pan-Archival Catalogue Services
+A message-based API for Pan-Archival Catalogue (PAC) services.
 
-A message-based API for the Pan-Archival Catalogue
+The PAC API is implemented using Cats Effect, to provide an asynchronous runtime, and an asynchronous   communications protocol (JMS). The reason for using an asynchronous protocol, rather than a synchronous one, like HTTP, is to ensure strong decoupling between the service and client systems. Strongly decoupled systems are much less likely to experience cascading failures, where the failure of one system causes others to fail, and are much less likely to become overloaded by too many requests.
+
+**Cats Effect Services**
+
+![alt text][logo]
```

Comment:
Logo is missing.

Reply:
Is it `images/Cats_Effect_Services.png`?

Reply:
Yes.

Reply:
I'm afraid it's still not showing correctly:

<img width="936" alt="image" src="https://user-images.githubusercontent.com/119881905/222407748-5e83ccf4-369f-4ff5-8261-ad21f62d6de1.png">

Reply:
This is what you'll need:

```
![Cats Effect Services](images/Cats_Effect_Services.png)  
```

Reply:
Thanks. That seems to work.

Reply:
Confirmed.